requirements-docker.txt

```diff
@@ -2,7 +2,7 @@ numpy
 pystan
 scikit-learn
 scipy<1.3  # >1.3 to use more recent pmdarima
-pmdarima  # <1.3 if scipy is kept this way
+pmdarima<1.3 # >1.3 if scipy>1.3 is chosen
```

Comment:
hope this doesn't break something :). One more reason to be careful with setup of unit tests